django/db/models/functions/comparison.py

```diff
@@ -164,7 +164,11 @@ def as_native(self, compiler, connection, *, returning, **extra_context):
         class ArgJoiner:
             def join(self, args):
                 pairs = zip(args[::2], args[1::2], strict=True)
-                return ", ".join([" VALUE ".join(pair) for pair in pairs])
+                # By default 'Cast' will use the double colon notation on postgres
```

Comment:
My comment here is almost certainly too verbose. Let me know how I can condense it

Reply:
Do we need this? As far as I'm aware, skipping 81ccf92f154c6d9eac3e30bac0aa67574d0ace15 when server-side binding is uses should do the trick :thinking:

Reply:
Do we absolute need it with the other suggested change? No.

But is this a defensive change that will keep the behavior correct if there are other changes down the line? I would argue _yes_